### PR TITLE
Update resize function

### PR DESCRIPTION
**Pull request by yoalex5**
GitHub issue #124 

What were changed:
1. Fixed bug when a callback is not being called on a client side
2. A new method `AdViewUtils.findPrebidCreativeSize()` was created to support success and failure handlers 
3. The old resize function(`findPrebidCreativeSize(View, CreativeSizeCompletionHandler)`) was marked as deprecated and it's still possible to use it (to support compatibility)

Screen Shot 2019-07-04 at 6 57 56 PM 1

The latest API for resize function looks like

```
        dfpAdView.setAdListener(new AdListener() {
            @Override
            public void onAdLoaded() {
                super.onAdLoaded();

                AdViewUtils.findPrebidCreativeSize(dfpAdView, new AdViewUtils.PbFindSizeListener() {
                    @Override
                    public void success(int width, int height) {
                        dfpAdView.setAdSizes(new AdSize(width, height));

                    }

                    @Override
                    public void failure(@NonNull PbFindSizeError error) {
                        Log.d("MyTag", "error: " + error);
                    }
                });

            }
        });
```

Error codes:
```
UNSPECIFIED_CODE = 201;
UNSUPPORTED_ANDROID_IPI_CODE = 202;
COMPOSITE_FAILURE_CODE = 203;
NO_WEBVIEW_CODE = 210;
WEBVIEW_FAILED_CODE = 220;
NO_HTML_CODE = 230;
NO_SIZE_OBJECT_CODE = 240;
NO_SIZE_VALUE_CODE = 250;
SIZE_UNPARSED_CODE = 260;
```

What to do:
1. Update docs

### CODE REVIEW

**Comment by yoalex5** [comment]
To fix Travis CI build error, please merge #123 before

**Comment by avohraa** [comment]
The changes look fine, but my concern is do we need those changes? To me, it seems like the only required change was adding `completionHandler.onSize(null);` within the following condition:
```
if (webViewList.size() == 0) {
        LogUtil.w("adView doesn't include WebView");
        completionHandler.onSize(null);
        return;
}
```

**Comment by yoalex5** [comment]
I think that division into `success` and `failure` parts is a good practice. Especially when it is the API for clients. It will allow:
1. remove some logic (null check) from a client side
2. provide a user friendly error message via `CreativeSizeError`
3. have the same API as iOS has

By the way, recently we received a publisher's report to improve this feature. And I hope these changes will help them

**Comment by yoalex5** [comment]
Today I am going to update this PR

File: PrebidMobile/API1.0/src/main/java/org/prebid/mobile/Util.java [hunk 1]
```diff
@@ -57,200 +51,26 @@ private Util() {
...
-    static void iterateWebViewListAsync(@Size(min = 1) final List<WebView> webViewList, final int index, final WebViewPrebidCallback webViewPrebidCallback) {
-
-        final WebView webView = webViewList.get(index);
-
-        webView.evaluateJavascript("document.body.innerHTML", new ValueCallback<String>() {
-
-            private void repeatOrFail() {
-                int nextIndex = index - 1;
+    /**
+     *@deprecated Please migrate to - Util.findPrebidCreativeSize(View, CreativeSizeResultHandler)
```
[the remark this answers is not in the document]
Comment (yoalex5) [reply PrebidMobile/API1.0/src/main/java/org/prebid/mobile/Util.java]: @avohraa, thanks for review
I have fixed the mistake

**Comment by yoalex5** [comment]
@avohraa, could you please make a review

**Comment by yoalex5** [comment]
@anwzhang, @avohraa, could you please make a review

**Pull request merged**